doc/README.md

```diff
@@ -0,0 +1,8 @@
+
+# Nixpkgs/doc
+
+This directory houses the sources files for the Nixpkgs manual.
+
+You can find the [rendered documentation for Nixpkgs `unstable` on nixos.org](https://nixos.org/manual/nixpkgs/unstable/).
```

Comment:
add a link to the stable doc
`You can find the [rendered documentation for Nixpkgs `stable` on nixos.org](https://nixos.org/manual/nixpkgs/stable/).`

and maybe also a link to https://nixos.org/learn.html.

Reply:
done